### PR TITLE
add announcement to teacher dashboard to sign petition

### PR DESCRIPTION
**Pull request by tanyaparker**
image

### CODE REVIEW

**Comment by tanyaparker** [comment]
I'll add strings to i18n gsheet, and then sync to all locales after your review.

File: pegasus/cache/i18n/en-US.yml [hunk 1]
```diff
@@ -665,6 +665,8 @@
   dashboard_announce_view_all: "View all"
   dashboard_announce_message: "<b>April 22, 2016</b> - Unplugged activities are a vital part of the learning process, but teachers tell us that finding all of the resources can be confusing and time-consuming. We listened! Now, you can find all of the unplugged resources on a single landing page. Associate the lessons with any computer science class, or use stand-alone as an independent course."
   dashboard_announce_button_text: "Go unplugged"
+  dashboard_announce_petition: "As a teacher, you're leading the way in a movement to give every student in every school the opportunity to learn computer science. Now, it's time to let your representatives hear from you. This week, education leaders, 28 governors, and CEOs of America's largest companies joined together in an open letter asking U.S. Congress to fund K-12 computer science education."
```
Comment (breville) [inline pegasus/cache/i18n/en-US.yml:668]: Is it "the" U.S. Congress?

File: pegasus/sites.v3/code.org/views/teacher_announcements.haml [hunk 2]
```diff
@@ -16,3 +17,12 @@
     %div{style: "float: right; margin-top: 25px;"}
       %a{:href=>cta_url}
         %button= I18n.t(:dashboard_announce_button_text)
+  %div.announce-content
+    %a{:href=>petition_url}
+      %div.content-img-container
+        %img.content-img{:src=>"/images/fill-300x200/teacher-dashboard/petition-faces.png"}
```
Comment (breville) [inline pegasus/sites.v3/code.org/views/teacher_announcements.haml:23]: +1 for using a folder under /images/

**Comment by breville** [comment]
lgtm

File: pegasus/cache/i18n/en-US.yml (hunk 1 above)
Comment (tanyaparker) [inline pegasus/cache/i18n/en-US.yml:668]: Hm yeah I like that better.

**Pull request merged**